### PR TITLE
WorkerSliceGWRecycler CRD creation

### PR DESCRIPTION
**Pull request by PuranAdhikari**

### CODE REVIEW

File: service/kube_slice_resource_names.go [hunk 1]
```diff
@@ -184,6 +185,11 @@ var (
 			APIGroups: []string{apiGroupKubeSliceWorker},
 			Resources: []string{resourceWorkerSliceConfig, resourceWorkerSliceGateways, resourceWorkerServiceImport},
 		},
+		{
```
Comment (PuranAdhikari) [inline service/kube_slice_resource_names.go:188]: Add same verbs in status as well if you need.

Comment (PuranAdhikari) [reply service/kube_slice_resource_names.go:188]: Also, if UI needs to perform actions on the CR, then add rules in `readOnlyRoleRules` and `readWriteRoleRules` as well.

**Review by PuranAdhikari — COMMENTED** [review]
Also, annotations need to be added in `main.go` and do `make manifests`

File: service/kube_slice_resource_names.go [hunk 2]
```diff
@@ -41,6 +41,7 @@ const (
 	resourceSecrets              = "secrets"
 	resourceEvents               = "events"
 	ResourceStatusSuffix         = "/status"
+	resourceWorkerSliceGwRecycler = "workerslicegwrecyclers"
```
Comment (narmidm) [inline service/kube_slice_resource_names.go:44]: put after line 40

**Comment by narmidm** [comment]
Only Sign Commit allowed. Verified.

**Comment by narmidm** [comment]
Update main.go fro controller permission

**Pull request merged**